code/backend/pages/01_Ingest_Data.py

```diff
@@ -29,6 +32,24 @@
 st.markdown(mod_page_style, unsafe_allow_html=True)
 
 
+def reprocess_all():
+    if env_helper.AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION:
+        reprocess_integrated_vectorization()
+    else:
+        remote_convert_files_and_add_embeddings()
```

Comment:
Can we extract this if-else into a utility class outside of the Ingest page?

Reply:
Moved it to backend function